src/clj_kondo/impl/analyzer.clj

```diff
@@ -448,6 +450,28 @@
           :list exprs
           (recur rest-exprs))))))
 
+(defn extract-arity-info [ctx parsed-bodies]
+  (reduce (fn [acc {:keys [:fixed-arity :varargs? :min-arity :ret :args :arg-vec]}]
+            (let [arg-tags (when (some identity args)
+                             args)
+                  v (assoc-some {}
+                                :ret ret :min-arity min-arity
+                                :args arg-tags)]
+              (if varargs?
+                (assoc acc :varargs v)
+                (do
+                  (when (get acc fixed-arity)
+                    (findings/reg-finding! ctx
+                                           (node->line
+                                            (:filename ctx)
+                                            arg-vec
+                                            :conflicting-fn-arity
+                                            (format "More than one function body with arity %s."
```

Comment:
let's replace `body` with `overload` since body usually refers to the thing after the argument vector, see
https://github.com/clojure/core.specs.alpha/blob/7d67e2f863ea28406f03338a7b92e2026d555062/src/main/clojure/clojure/core/specs/alpha.clj#L79